bin/cli.js

```diff
@@ -18,6 +18,8 @@ const cliOptions = [
   [ '--node-only <patterns>',    'Node only tests patterns', splitOpts, [] ],
   [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
   [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '-v, --verbose',             'Verbosity', (val, acc = 0) => acc + 1],
+  [ '-q, --quiet',               'Suppress output' ],
```

Comment:
This option contradicts the `--verbose` option; I don't think it should be allowed to provide them both at the same time when running the CLI.